editor.planx.uk/src/@planx/components/AddressInput/AddressInput.stories.tsx

```diff
@@ -5,27 +5,32 @@ import React from "react";
 import Wrapper from "../fixtures/Wrapper";
 import Editor from "./Editor";
 import Public from "./Public";
+import AddressInputComponent from "./Public";
```

Comment:
nit: `AddressInputComponent` should be the same as `Public` on the line above in this case? We deal with this a bit inconsistently across Public component files and test/story files, so happy to go with whichever you think is most clear here as long as we only import/use one or the other!

Reply:
Thanks for spotting this - you're right and I'll remove this duplicated import.

I think `import Public from "./Public";` will be the more consistent pattern here.